README.md

```diff
@@ -1,3 +1,5 @@
+Personal Development Project on RIOT OS  (see branch feedback-scheduling -> examples -> RRTester)
+
```

Comment:
this shoul prbably be part of the PR reques message ( not a change to the RIOT-OS readme)